NEWS.md

```diff
@@ -1,5 +1,12 @@
 # Unreleased
 
+## Breaking changes
+
+- `set_image_path()`, `set_image_url()`, and `set_image_webshot()` have been
+  deprecated and will be removed in a future update. They have been replaced by
+  `set_content_image(content, path)`, which works both with local file paths and
+  remote URLs to images.
```

Comment:
It might be worth adding in here that `_webshot` is not being replaced, instead see the recipe at: ...

Reply:
Yeah, I'll update this PR when I have created a `connect` PR with the recipe URL, which I'll do before this merges.

Reply:
Hmm, if I link to the recipe, would this need to wait until Connect is released? 🤔